tests/testthat/test_flexsurv_surv_flexible.R

```diff
@@ -1,29 +1,34 @@
 test_that("autotest", {
-  set.seed(200)
-  learner = lrn("surv.flexible", k = 1, scale = "normal", inits = c(0.1, 1))
-  expect_learner(learner)
-  # sanity excluded due to issues in optimisation (it's too easy...)
-  result = run_autotest(learner, check_replicable = FALSE, exclude = "sanity")
-  expect_true(result, info = result$error)
+  with_seed(200, {
```

Comment:
one way to not have this nested code is to use `local_seed()` but this is just aesthetics so I don't care, just wanted to let you know :)